app/views/repair_requests/new.html.erb

```diff
@@ -86,3 +88,13 @@
     </div>
   </div>
 </div>
+
+<script>
+  var descriptionText = document.getElementById("description-text");
+  var counter = document.getElementById("counter");
+  var maxLength = counter.dataset.maximumLength;
+
+  descriptionText.addEventListener("keyup", function() {
```

Comment:
When you move this to a .js file, you'll also need to change this into a `onkeyup=` attribute on the textarea.

I'm not sure if this is a good practice, but it's being used everywhere in the code, so we should stick with it unless we find a good reason not to.

Reply:
Yep, changing it now